### ISSUE TITLE
App context duplicated in navLink

### ISSUE THREAD

**Issue by tdupard** [TheCoder4eu/BootsFaces-OSP#200]
```xml
<b:navbarLinks>
        <b:navLink value="Home" href="#{request.contextPath}/index.xhtml?faces-redirect=true" />
```
Renders this:
         http://localhost:9080/myapp/myapp/index.xhtml?faces-redirect=true

**Comment by stephanrauh** [comment]
I'm not sure that's anything to do with BootsFaces. Mind you, the #{request.contextPath} variable isn't resolved by BootsFaces, but by the underlying JSF engine (Mojarra or MyFaces).

I suggest you try to things:
* First, please put the  #{request.contextPath}  outside the navLink, the idea being that it's simply printed on the screen:
```xml
#{request.contextPath}
<b:navbarLinks>
        <b:navLink value="Home" href="#{request.contextPath}/index.xhtml?faces-redirect=true" />
```
I suspect you'll see the duplicate context, too.
* Second, check your web.xml. Does it contain multiple URL-patterns? My current recommendation is to use one and only one URL pattern. (I know our showcase recommends something different, but that's going to change soon). 
```xml
 <servlet-mapping>
     <servlet-name>Faces Servlet</servlet-name>
    <url-pattern>*.jsf</url-pattern>
</servlet-mapping>
```

**Comment by stephanrauh** [comment]
I managed to reproduce the error. It occurs in line 297 of NavLink.java. @TheCoder4eu Riccardo, can you have a look at it? I suspect calling `fc.getApplication().getViewHandler().getResourceURL(fc, value);` is superfluous.

**Comment by TheCoder4eu** [comment]
I have to do some tests, maybe this call is superfluous for this specific case but not for other values for href.

**Comment by TheCoder4eu** [comment]
Should be solved in v0.8.0
